Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/SpeechCommands.cs

```diff
@@ -6,26 +6,44 @@
 
 namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem
 {
+    /// <summary>
+    /// Speech command definition, defining each keyword, key code and it's corresponding logical Action the command will perform.
+    /// </summary>
     [Serializable]
     public struct SpeechCommands
     {
+        /// <summary>
+        /// Speech commands constructor
+        /// </summary>
+        /// <param name="keyword">Word to be recognized</param>
+        /// <param name="keyCode">Corresponding keyboard input</param>
+        /// <param name="action">Logical action to be performed by the Input System</param>
         public SpeechCommands(string keyword, KeyCode keyCode, InputAction action)
         {
             this.keyword = keyword;
             this.keyCode = keyCode;
             this.action = action;
         }
 
+        /// <summary>
+        /// The input string keyword that will be listened for by the SpeechInputSource
+        /// </summary>
```

Comment:
Summary tags in wrong place.

Also let's do a doc pass in a separate PR.

I just wanna see the controller implementation.